packages/trader/src/Modules/Trading/Components/Form/screen-small.tsx

```diff
@@ -108,9 +124,7 @@ const CollapsibleTradeParams = ({
                     <Strike />
                 </div>
             )}
-            {/* 
-            // @ts-expect-error Observer wrapped component needs to be ts migrated before props can be detected */}
-            {!is_accumulator && <MobileWidget is_collapsed={is_collapsed} toggleDigitsWidget={toggleDigitsWidget} />}
+            {!is_accumulator && <MobileWidget />}
```

Comment:
In `<MobileWidget />` `is_collapsed` and `toggleDigitsWidget` props were used only for old Digit Widget. As it was removed (lines 93-105) there is no need in them anymore